apps/server/src/shared/domain/entity/base.entity.ts

```diff
@@ -29,10 +29,10 @@ export abstract class BaseEntityWithTimestamps<Optional = never> implements Auth
 	@SerializedPrimaryKey()
 	id!: string;
 
-	@Property()
+	@Property({ type: 'date', nullable: true, onCreate: () => new Date() })
```

Comment:
Is it still necessary to initialize the property statically like `createdAt = new Date();`?

Reply:
According to the documentation of microORM it is: https://mikro-orm.io/docs/3.6/defining-entities